addons/mass_mailing_hr/views/hr_employee_views.xml

```diff
@@ -0,0 +1,13 @@
+<odoo>
+    <record id="action_employee_mass_mail" model="ir.actions.server">
```

Comment:
A whole bridge module for a server action ? Meh ... each bridge module adds latency (registry loading, ...). It has a non negligible impact on performances, hence... meh.

I understand it is practical, but maybe we should investigate a generic way of adding it in the cog menu (this time, I think some JS tweak is the way to go: if model is _mailing_enabled, add an entry to send a mailing on this model in the generic action menu.

cc @awa-odoo for followup (and js)